lib/projects/index.ts

```diff
@@ -74,7 +61,7 @@ export async function getProjectConfig(dir?: string): Promise<{
   projectDir: string | null;
   projectConfig: ProjectConfig | null;
 }> {
-  const configPath = await getProjectConfigPath(dir);
+  const configPath = getProjectConfigPath(dir);
```

Comment:
From what I can tell, `getProjectConfig` doesn't actually need to be async. But I think I've probably already passed my scope creep limit for this PR 😅